### ISSUE TITLE
Use input number instead of float value

### ISSUE THREAD

**Issue by thomasoggier** [ScratMan/HASmartThermostat#274]
Il would like to use variable value for fixed value in the yaml file. For sample, I would like to change output_min.
So that I no longer need to restart hassio when a value changes

**Comment by Chupaka** [comment]
What's the use case for that?

Generally, you can use Automations to translate output value to your device(s)

**Comment by thomasoggier** [comment]
I would like to be sure that the floor of my bathrooms remains warm, so that there is always a minimum of heating. On the other hand, if the temperature is below a certain temperature the PID takes over. The problem is that I do not yet know what value I must set so that the floor remains warm, without overheating

**Comment by thomasoggier** [comment]
Sorry but first, thanks for your answer

**Comment by Chupaka** [comment]
Did you try restarting smart thermostat alone ("smart_thermostat: reload" action in Developer Tools) without restarting Home Assistant?

**Comment by thomasoggier** [comment]
Yes, it works perfectly, but I found it interesting to be able to have a graphic view directly from my cards.

**Comment by Chupaka** [comment]
Unfortunately, SmartThermostat doesn't expose output_min/max params in a state, so you can't even see them in GUI
